src/Tribe/Commerce/PayPal/Main.php

```diff
@@ -1571,6 +1571,9 @@ public function get_ticket( $event_id, $ticket_id ) {
 		$return->end_time         = get_post_meta( $ticket_id, '_ticket_end_time', true );
 		$return->sku              = get_post_meta( $ticket_id, '_sku', true );
 
+		// All tickets are 'none' by default, ET+ will filter to change.
+		$return->iac = 'none';
```

Comment:
This really should be a constant from the Abstract, so we avoid having the string being passed around.

Reply:
I set it on the ticket object, can remove this

Reply:
Removed